### ISSUE TITLE
Warning: Undefined array key "mfaEnrollmentId"

### ISSUE THREAD

**Issue by guillaumesmo** [kreait/firebase-php#783]
### Describe the bug

When 2FA is enabled in Firebase Auth, and a user has enrolled a 2FA method, Auth::listUsers and Auth::getUser throw the following exception

```
ErrorException: "Warning: Undefined array key "mfaEnrollmentId"" at .../vendor/kreait/firebase-php/src/Firebase/Auth/MfaInfo.php line 42
```

this seems to be caused by the API returning an array of MfaInfo, while the php lib expects only one

I was able to fix with the following changes:
https://github.com/kreait/firebase-php/compare/7.x...guillaumesmo:firebase-php:7.x

### Installed packages

```json
"kreait/firebase-php": "^7.0",
```
```


### PHP version and extensions

```shell
php                    8.2.3
```

### Steps to reproduce the issue.

```php
$auth->getUser($userId);
```

### Error message/Stack trace

```shell
ErrorException: "Warning: Undefined array key "mfaEnrollmentId"" at .../vendor/kreait/firebase-php/src/Firebase/Auth/MfaInfo.php line 42
```

**Comment by jeromegamez** [comment]
Thanks for reporting this - I had implemented this to resolve #749, but I wasn't able to test this properly (and @jesperbjerke, who requested it, might not have either 😅).

As far as I can see (from my phone) I can/would not change the existing property as it would be a breaking change, and introduce a new property instead (perhaps `mfaEnrollments`).

Can I rely on you properly testing this for me once I create the according PR? 🙏🏻

**Comment by guillaumesmo** [comment]
Firebase supports only phone MFA, so I don't think it's needed to add another property for now as it will only contain the single item which is already there. Instead the property can be passed the only item if present, or null

And yes, I'm able to test this for you

**Comment by jeromegamez** [comment]
Could you give #784 a try?

**Comment by guillaumesmo** [comment]
Hello,

tested:
```
kreait/firebase-php                   dev-mfa-info 21b918b
```

works as expected!

thank you very much

**Comment by jeromegamez** [comment]
Thank you 🙏🏻! I will then hopefully merge and release it later this evening, tomorrow at latest!

**Comment by jeromegamez** [comment]
Fixed with 8d82198045bafd977bde1dc2e8283284129d2d8b

**Comment by guillaumesmo** [comment]
Thank you for the fix. Awesome package!
